tensorboard/main.py

```diff
@@ -279,9 +272,26 @@ def ConstructDebuggerPluginWithGrpcPort(context):
         projector_plugin.ProjectorPlugin,
         text_plugin.TextPlugin,
         profile_plugin.ProfilePlugin,
-        ConstructDebuggerPluginWithGrpcPort,
     ]
 
+    def ConstructDebuggerPluginWithGrpcPort(context):
+      try:
+        # pylint: disable=line-too-long,g-import-not-at-top
+        from tensorboard.plugins.debugger import debugger_plugin as debugger_plugin_lib
+        # pylint: enable=line-too-long,g-import-not-at-top
+        tf.logging.info("Starting Debugger Plugin at gRPC port %d",
+                        FLAGS.debugger_data_server_grpc_port)
+        debugger_plugin = debugger_plugin_lib.DebuggerPlugin(context)
+        debugger_plugin.listen(FLAGS.debugger_data_server_grpc_port)
+        return debugger_plugin
+      except ImportError:
+        raise ImportError(
```

Comment:
Can we print this and then reraise the original error (just `raise`), to prevent confusion in case the `ImportError` is _not_ caused by futures or grpcio?

Similarly, can we move everything except for the `import` statement out of the `try` block?
```py
try:
  from … import …
except:
  raise …
tf.logging.info(…)
…
```